lib/Sema/PlaygroundTransform.cpp

```diff
@@ -786,6 +798,15 @@ class Instrumenter : InstrumenterBase {
     return std::make_pair(PBD, VD);
   }
 
+  bool shouldLog(ASTNode node) {
+    // Don't try to log ~Copyable types, as we can't pass them to the generic logging functions yet.
+    if (auto *VD = dyn_cast_or_null<ValueDecl>(node.dyn_cast<Decl *>()))
+      return VD->hasInterfaceType() ? !VD->getInterfaceType()->isNoncopyable() : true;
```

Comment:
Don’t call hasInterfaceType(), it doesn’t do what you think it does (it means “have I computed the type yet” not “this has a type”)

Reply:
Yes that looks good, thanks!